application/views/admin/token/csvimportresult.php

```diff
@@ -101,6 +102,24 @@
                                 </li>
                             <?php } ?>
 
+                            <?php if (!empty($aPluginErrorMessageList)) { 
+                                        $iPluginErrorIndex=0;
+                                        foreach ($aPluginErrorMessageList as $sPluginErrorMessage => $aTokenSpecificErrorList) { 
+                                            $iPluginErrorIndex++; ?>
```

Comment:
Kind of too much logic in a viewfile, but not sure how to fix. Maybe do a small widget? Or too much work?

Reply:
The only difference in between previous blocks and this new one is we are just numbering lines... I wouldn;t do a widget as it is not reusable  If not, I could add some logic in the top of the file as to add the numbering and create a plain array. What do you think?

Reply:
Hm, such logic should preferably be in the controller. Bah. Yes, it's not an easy task to separate view from logic. Leave for now and concentrate on the event function above instead.